test/powershell/Modules/PackageManagement/PackageManagement.Tests.ps1

```diff
@@ -13,16 +13,21 @@
 #
 # ------------------ PackageManagement Test  -----------------------------------
 
-$InternalGallery = "https://www.poshtestgallery.com/api/v2/"
-$InternalSource = 'OneGetTestSource'
+$gallery = "https://www.powershellgallery.com/api/v2"
+$source = 'OneGetTestSource'
 
 Describe "PackageManagement Acceptance Test" -Tags "Feature" {
 
  BeforeAll{
     Register-PackageSource -Name Nugettest -provider NuGet -Location https://www.nuget.org/api/v2 -Force
 
-    ## Comment out this line because 'poshtestgallery.com' is down, tracked by https://github.com/PowerShell/PowerShell/issues/17019
-    ## Register-PackageSource -Name $InternalSource -Location $InternalGallery -ProviderName 'PowerShellGet' -Trusted -ErrorAction SilentlyContinue
+    $packageSource = Get-PackageSource -Location $gallery -ErrorAction SilentlyContinue
+    if ($packageSource) {
+        $source = $packageSource.Name
```

Comment:
If it's already registered, ensure it has `Trusted` property for repository set to true. Based on test error, it still considers this an untrusted repository.
```suggestion
        $source = $packageSource.Name
        Set-PackageSource -Name $source -Trusted
```

Reply:
Fixed. Thanks!